### PR TITLE
Add G Desktop Suite v0.2.1

### PR DESCRIPTION
**Pull request by alexkim205**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by miccal** [comment]
```
==> Installing node
==> Pouring node-14.2.0.catalina.bottle.tar.gz
==> Finishing up
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink bin/node
Target /usr/local/bin/node
is a symlink belonging to node@12.
```
I don't think this is an issue, since for most users `node` is symlinked to `/usr/local` while `node@12` (and incidentally`node@10`) is not:
```
-bash-5.0.17- /Users/miccal (30) [> brew info node
node: stable 14.2.0 (bottled), HEAD
Platform built on V8 to build network applications
https://nodejs.org/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/node.rb
==> Dependencies
Build: pkg-config ✘, python@3.8 ✔
Required: icu4c ✔
==> Options
--HEAD
	Install HEAD version
```
```
-bash-5.0.17- /Users/miccal (30) [> brew info node@12
node@12: stable 12.16.3 (bottled) [keg-only]
Platform built on V8 to build network applications
https://nodejs.org/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/node@12.rb
==> Dependencies
Build: pkg-config ✘, python@3.8 ✔
Required: icu4c ✔
==> Caveats
node@12 is keg-only, which means it was not symlinked into /usr/local,
because this is an alternate version of another formula.
```
```
-bash-5.0.17- /Users/miccal (32) [> brew info node@10
node@10: stable 10.20.1 (bottled) [keg-only]
Platform built on V8 to build network applications
https://nodejs.org/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/node@10.rb
==> Dependencies
Build: pkg-config ✘
Required: icu4c ✔
==> Requirements
Required: macOS is required ✔
==> Caveats
node@10 is keg-only, which means it was not symlinked into /usr/local,
because this is an alternate version of another formula.
```

File: Casks/gsuite.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'gsuite' do
+  version '0.2.1'
+  sha256 '6220c88551ab2c19180d662387330ab31e44a45cb5dabad51bb86a0bf47904eb'
+
+  # github.com/fitztrev/shuttle was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/gsuite.rb:5]: This is incorrect. Remove.

File: Casks/gsuite.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'gsuite' do
```
Comment (vitorgalvao) [inline Casks/gsuite.rb:1]: `g-desktop-suite`

File: Casks/gsuite.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  version '0.2.1'
+  sha256 '6220c88551ab2c19180d662387330ab31e44a45cb5dabad51bb86a0bf47904eb'
+
+  # github.com/fitztrev/shuttle was verified as official when first introduced to the cask
+  url "https://github.com/alexkim205/G-Desktop-Suite/releases/latest/download/G.Desktop.Suite-#{version}.dmg"
+  appcast 'https://github.com/alexkim205/G-Desktop-Suite/releases.atom'
+  name 'G Desktop Suite'
+  homepage 'https://github.com/alexkim205/G-Desktop-Suite'
+
+  depends_on formula: 'node'
```
Comment (vitorgalvao) [inline Casks/gsuite.rb:11]: If this is an app built with electron, why would `node` be required?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Thank you for the fast fixes. Welcome.

**Pull request merged**